### PR TITLE
Beta 2023.05.22

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 1]
```diff
@@ -276,7 +276,7 @@ Mario-kun. She just clicked her tongue behind your back.
 -- 「オレたちの仕事は吸血鬼掃除だ。ただし標的は真祖……アルクェイドじゃねえ。厄介な競争相手だが、アレと正面からやり合うほど蛮勇じゃねえからな。
 "Our job's to eliminate vampires.
  But our target %{i}ain't%{/i} the True Ancestor――Arcueid.
- She's an annoying-ass competitor, but I ain't dumbass enough to fight somethin' like her straight up.
+ She's an annoying-ass competitor, but I ain't dumb enough to fight somethin' like her straight up.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:279]: Might keep this one, it's Mario flair

File: script/Arcueid/Day 14/14_00_ARC14_1.txt [hunk 2]
```diff
@@ -223,7 +223,7 @@ Every now and then, a memory resurfaces.
 [sha:eca07110967d85830c7938369cb0e49f2da5c088]{
 -- Page 19, Offset 36790.
 -- この思いに耐えきれずどうにかなってしまうのか、それとも、それにさえ慣れて空虚なまま過ごすのか。
-I wonder if some day I won't be able to bear it anymore and will go insane,
+I wonder if some day I won't be able to bear it anymore and go insane,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 14/14_00_ARC14_1.txt:226]: Doesn't feel like a correct elision to me

File: script/Arcueid/Day 8/08_00_ARC08_8.txt [hunk 3]
```diff
@@ -266,7 +266,7 @@ I accurately predict and parry each swing of the blade with my knife.
 -- Page 500, Offset 21001.
 -- この闇の中で、休む間もなく振るわれる凶器を、的確に止めている自分自身の体だった。
 Despite the darkness,
- despite the relentless onslaught of that deadly weapon,
+ and the relentless onslaught of that deadly weapon,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 8/08_00_ARC08_8.txt:269]: Prefer the repetition here personally

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 4]
```diff
@@ -476,7 +476,7 @@ When you say 'opening act', which part was that exactly... sir?"
 -- 逃げ足だけは見所あるよ、オマエ。10年近く使い捨ての狗ころやってるヤツは筋金入ってやがる。オレが法王庁に帰り次第、テメェは首だ。修道院送りじゃねえぞ。ギロチン送りの方な」
 // 首を切る meaning to fire someone and to behead someone.
 Only thing you're any goddamn good at is running away.
- You'd think that after being a disposable attack dog for ten years you'd have grown a fucking backbone.
+ You'd think that after being a disposable attack dog for ten years, you'd have grown a fucking backbone.
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:479]: Doesn't sit right to me, first part isn't really a complete thought

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 5]
```diff
@@ -445,7 +445,8 @@ Ever since she became an Executor,
 -- Page 404, Offset 23177.
 -- わたしが聞いた中で一番長かったのは、たった一人の死徒を殺すために一年間、その街の下水路に潜伏していたって話」
 From what I've heard, the longest she ever spent on a mission was an entire year.
- She spent the whole time hiding in the sewers under a city, all to take down just a single Dead Apostle."
+ She hid in the city sewers the whole time,
```
Comment (rschlaikjer) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:448]: Prefer original. Better link on the temporality.

File: script/Arcueid/Day 10/10_00_ARC10_6.txt (hunk 1 above)
Comment (Petrikow) [reply script/Arcueid/Day 10/10_00_ARC10_6.txt:279]: yeah, keep it

File: script/Ciel/Day 9/09_01_CIEL09_4.txt (hunk 5 above)
Comment (Petrikow) [reply script/Ciel/Day 9/09_01_CIEL09_4.txt:448]: maybe "under" is a bit redundant, at least

sewers of a city

**Pull request merged**